packages/frameworks/src/frameworks.ts

```diff
@@ -554,6 +554,94 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'SolidStart',
+    slug: 'solidstart',
+    demo: 'https://solidstart.examples.vercel.com',
+    logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/solid.svg',
+    tagline:
+      'A Reactive JavaScript Library for Building User Interfaces.',
+    description: 'A Solid app, created with SolidStart.',
+    website: 'https://solidjs.com',
+    envPrefix: 'VITE_',
```

Comment:
Should this be `SOLID_` since this framework isn't using vite?

Reply:
It also uses Vite which brings up an interesting question if the detection rules are adequate since Vite and Solid will be present for both templates. Solid Start just buries the Vite commands and uses the programmatic APIs instead for setting defaults and better control but the environment and bundling is all still Vite.

Reply:
It should evaluate in order so the first match is used for the framework.

However, I'm wondering if we really need both here. 

Why are there two options if they both have the same dependencies? Shouldn't the framework using a single build command and then provide config to change internals. Like for Next.js we use `next.config.js` to let the user swap Babel for SWC for example.

Reply:
They don't completely have the same dependencies just they both use Solid and Vite. One is just standard Vite template set up for client rendered SPAs, with all Vites standard things like index.html in the root of the project.

The other tries to take a more opinionated approach and has a completely different entry setup, basically setup for SSR(it hydrates from the Document root, something that you can do with a client side render), has a other features like built in file-based routing, head components etc, and has a CLI wrapper over Vite to do custom builds for different adapter platforms and honestly is a work in progress still.

It might be easiest to think of it as the difference between Svelte and SvelteKit or Vue and Nuxt. We just happen to use Vite as the bundler for both. 

I guess my question is around dependencies. The project will have their own dependencies and the build script should install them, but what does the dependency field in the framework.ts do? What needs to go there? I took it to mean to install the CLI. And for the basic Vite template I put `vite` there. But SolidStart has peer deps that it expects to come in as part of the repo since it is more than just the CLI but also the runtime/tooling parts. Do those need to go in the dependency list as well? Or is that for something else?

Reply:
> One is just standard Vite template set up for client rendered SPAs, with all Vites standard things like index.html in the root of the project.

In that case, we don't need an entry for it since we already have Vite.

Reply:
> what does the dependency field in the framework.ts do

Its legacy before `detectors` existed.

Reply:
> In that case, we don't need an entry for it since we already have Vite.
Ok sure I can remove the entry.  Vercel recognizing the project as Vite is fine from a detection standpoint. Should I keep/remove the example? I wasn't sure if people used this to find templates, where they are like "Hey I want to start a SolidJS project" and choose it.